### ISSUE TITLE
Any plan to combine this with diffuser?

### ISSUE THREAD

**Issue by Shengguang-Zhou** [ali-vilab/AnyDoor#46]
Does this have any plan or how to use it by diffuser? 

Also, does this support SDXL-Turbo?

**Comment by XavierCHEN34** [comment]
Yes, currently we have already started to develop a stronger version based on diffusers.

**Comment by Shengguang-Zhou** [comment]
> Yes, currently we have already started to develop a stronger version based on diffusers.  

Can't wait for it! I can help around as it's such an amazing project. Also, are we going to combine it with SDXL-Turbo?

**Comment by XavierCHEN34** [comment]
Thank you for your interest.  We also expect  a one/few step version and we would try it, however, this might requires some explorations.

**Comment by alexandrewillame** [comment]
Hello,

Is there any support planned for SDXL as well?

Thank you!
